lib/warnings.js

```diff
@@ -21,4 +21,6 @@ warning.create('FastifyDeprecation', 'FSTDEP010', 'Modifying the "reply.sent" pr
 
 warning.create('FastifyDeprecation', 'FSTDEP011', 'Variadic listen method is deprecated. Please use ".listen(optionsObject)" instead. The variadic signature will be removed in `fastify@5`.')
 
+warning.create('FastifyDeprecation', 'FSTDEP012', 'You are setting forceCloseConnections to "idle" but you server does not support "closeIdleConnections". The option has been changed to "true".')
```

Comment:
It is a warning, but not a `deprecation` warning.
Maybe `FastifyWarning` and `FSTWRN001` ?

Reply:
Good spot and suggestion.

Reply:
Yup, as Matteo also suggested, I'll throw instead. Will be in the next push!